src/node_internals.h

```diff
@@ -785,6 +786,13 @@ static inline const char *errno_string(int errorno) {
     TRACING_CATEGORY_NODE "." #one ","                                        \
     TRACING_CATEGORY_NODE "." #one "." #two
 
+extern void EnableTracingCategories(
+    const std::vector<std::string>& categories);
+extern void DisableTracingCategories(
+    const std::vector<std::string>& categories);
+extern const std::set<std::string>& GetEnabledTracingCategories();
+extern void StopTracing();
```

Comment:
You can drop `extern`, it doesn’t make a difference for function declarations + we don’t use it elsewhere